src/rules/color-hex-length/index.js

```diff
@@ -0,0 +1,48 @@
+import {
+  report,
+  ruleMessages,
+  styleSearch
+} from "../../utils"
+
+export const ruleName = "color-hex-length"
+
+export const messages = ruleMessages(ruleName, {
+  expected: (l, h) => `Expected ${l} hex color ${h}`,
```

Comment:
> Does that make sense? 

It does, and I think it'll make a great addition :)

We should do a quick audit of the messages in the other rules to make sure they also, where appropriate, behave the same.